easybuild/easyconfigs/d/DB/DB-18.1.25-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,31 @@
+# Contribution from the Crick Scientific Computing Team
+# uploaded by J. Sassmannshausen
+
+name = 'DB'
+version = '18.1.25'
+
+homepage = 'http://www.oracle.com/technetwork/products/berkeleydb'
+
+description = """Berkeley DB enables the development of custom data management
+ solutions, without the overhead traditionally associated with such custom
+projects."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+# download via
+# http://www.oracle.com/technetwork/products/berkeleydb/downloads/,
```

Comment:
@sassy-crick  I didn't see version 18.1.25 listed at http://www.oracle.com/technetwork/products/berkeleydb/downloads/ 
nor at https://www.oracle.com/database/technologies/related/berkeleydb-release-history.html
but the direct link https://download.oracle.com/otn/berkeley-db/db-18.1.25.tar.gz works (after sign-in), 
include this in the comment?